apps/backend/src/libs/modules/s3/libs/types/s3-settings.type.ts

```diff
@@ -0,0 +1,8 @@
+type S3Settings = {
```

Comment:
Do we need a separate type for this? You can define `type Constructor` in the module